_build/requirements.txt

```diff
@@ -14,3 +14,5 @@ pexpect>=4.5
 python-daemon
 pyyaml
 six
+pykerberos
+pywinrm
```

Comment:
`pywinrm` is already up on line 11.

Reply:
@shanemcd hi, not done this alot in github. Do you merge or do I need tochange my pull? :p

Reply:
Looks good, will merge after CI passes, looks like a fluke that it failed.